code/modules/mob/inventory.dm

```diff
@@ -44,6 +46,8 @@
 /mob/proc/put_in_r_hand(obj/item/W)

```

Comment:
all these procs shouldn't be at the /mob level, this is stupid. But it'd require a big mob refactor.